src/bindings/js/node/tests/basic.test.js

```diff
@@ -20,6 +20,31 @@ it('Core.getAvailableDevices()', () => {
     assert.ok(devices.includes('CPU'));
 });
 
+describe('Core.getVersions()', () => {
+  const devices = core.getAvailableDevices();
+
+  it('getVersions(validDeviceName: string)', () => {    
+    const deviceVersion = core.getVersions(devices[0]);
+    assert.ok(Object.keys(deviceVersion).length >= 1);
+  });
```

Comment:
I don't see any purpose to check it in this way. 
My proposals:
- check that `getVersions` returns object
- check that object contains property in specified format